### ISSUE TITLE
[Adaptive Card iOS - Flight Update]: Keyboard focus is going on the non-interactive elements while navigating using tab or shift+tab key.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#27]
**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.23.10.15.1)
Screen Reader: Voice Over
Keyboard: Logitech K480

**Pre-Requisite:**
Navigate through keyboard only.

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.5" control and select it.
5. Navigate to 'Scenario' control and activate it.
6. Navigate to 'FlightUpdate.JSON' and double tap to activate it.
7. The Card will open.
8. Press tab key and navigate the card and observe the issue.

**Actual Result:**
While navigating the Flight Update Card using tab or shift +tab key, keyboard focus is landing on the non-interactive elements.

**Expected Result:**
Keyboard focus should not land on the non -interactive elements while navigating the Flight Update card using tab or shift+tab key. It should be land only on interactive elements.

User Impact:
It will impact the keyboard user as they will try to perform action on non-interactive controls also due to this they have to put more effort to navigate on the cards.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/focus-order

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/39a9b45e-a136-49cf-8ea6-e19de9116ce4
